### PR TITLE
DOC : fix errors docstrings pandas.to_numeric

### PR DESCRIPTION
**Pull request by isoletslicer**
- [ ] closes #xxxx
- [ ] tests added / passed
- [ ] passes `black pandas`
- [x ] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [ ] whatsnew entry

added docstrings information about arg descriptions, downcast type 'int', and Return type

### CODE REVIEW

File: pandas/core/tools/numeric.py [hunk 1]
```diff
@@ -35,17 +35,23 @@ def to_numeric(arg, errors="raise", downcast=None):
...
+        - list is any kind of '[]' dtypes.
+        - tuple is any kind of '()' dtypes.
+        - 1-d array is list dtypes that only have 1 list.
+        - Series is kind of pandas 1-d ndarray with axis labels.
     errors : {'ignore', 'raise', 'coerce'}, default 'raise'
         - If 'raise', then invalid parsing will raise an exception.
         - If 'coerce', then invalid parsing will be set as NaN.
         - If 'ignore', then invalid parsing will return the input.
-    downcast : {'integer', 'signed', 'unsigned', 'float'}, default None
+    downcast : {'int', 'signed', 'unsigned', 'float'}, default None
```
Comment (MomIsBestFriend) [inline pandas/core/tools/numeric.py]: Please change this back to ```integer```, you are completely right when it comes to the type of rhe variable that we want the python type name and not the pre-historic name, but in this case it's the value, and code below will check if ```downcast``` is equals to ```integer``` and not for ```int```

File: pandas/core/tools/numeric.py [hunk 2]
```diff
@@ -35,17 +35,23 @@ def to_numeric(arg, errors="raise", downcast=None):
...
         If not None, and if the data has been successfully cast to a
         numerical dtype (or if the data was numeric to begin with),
         downcast that resulting data to the smallest numerical dtype
         possible according to the following rules:
 
-        - 'integer' or 'signed': smallest signed int dtype (min.: np.int8)
+        - 'int' or 'signed': smallest signed int dtype (min.: np.int8)
```
Comment (MomIsBestFriend) [inline pandas/core/tools/numeric.py]: Change this back as well

**Review by ShaharNaveh — CHANGES_REQUESTED** [review]

File: pandas/core/tools/numeric.py (hunk 1 above)
Comment (isoletslicer) [reply pandas/core/tools/numeric.py]: alright, ill be back soon

Comment (isoletslicer) [reply pandas/core/tools/numeric.py]: revert back to integer done, please kindly check :")

**Review by isoletslicer — COMMENTED** [review]
revert back to integer done, please kindly check :")

**Review by WillAyd — APPROVED** [review]

**Comment by WillAyd** [comment]
Thanks @isoletslicer

**Pull request merged**